### DISCUSSION TITLE
How to create a submenu vertical links for sidenav? Like atlassian style

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#41048] [Q&A]
Hi guys, I'd like to create something like this... anyone ever did it before? Thanks.

![Global+sidebar](https://user-images.githubusercontent.com/40239902/222815410-efb6cdba-9714-4f7c-805e-7ba0e1e0e0dc.png)

**Comment by kiner-tang** [comment]
Can you describe it more? Not sure what you want to do

**Reply by vitalijalbu** [reply]
I'd like to create a 2 level menu like in the photo, the 1st one as icon and the second level as links. Anyone did something similar before?

**Reply by vitalijalbu** [reply]
> Can you describe it more? Not sure what you want to do

Hi, is there any way to dothat? maybe 2 layouts isnide the Sider component?

**Reply by vitalijalbu** [reply]
DEMO: [https://cdn.dribbble.com/users/2173663/screenshots/17564399/media/bfe7626ee64863d544bd4376a7628225.jpg?compress=1&resize=1600x1200&vertical=top](https://cdn.dribbble.com/users/2173663/screenshots/17564399/media/bfe7626ee64863d544bd4376a7628225.jpg?compress=1&resize=1600x1200&vertical=top)
